useHonks.js

```diff
@@ -0,0 +1,24 @@
+export default function useHonks() {
+  let audio = null;
```

Comment:
Shouldn't this `audio` be shared between all hoo-- honks?

Reply:
honk!